kodilla-collections/src/main/java/com/kodilla/collections/interfaces/homework/Ford.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.collections.interfaces.homework;
+
+import java.util.Objects;
+
+public class Ford implements Car {
+
+    double speed;
+    private double acceleration;
+    private double deceleration;
+
+    public Ford(double speed, double acceleration, double deceleration) {
```

Comment:
Bez sensu miec takie same klasy., ktore sie roznia tylko nazwa. Parametry acceleration i deceleration powinny byc stale w kazdej z klas.

Reply:
Potem bedziesz mial konflikty

Reply:
W naszym zadaniu kazdy Ford bedzie taki sam. Zreszta w zyciu jest podobnie - jak kupujesz model danego samochodu, to kazdy ma takie same parametry (w odniesieniu do tego samego modelu i jego wersji). Zadanie jest uproszczeniem sytuacji z zycia i zakladamy, ze tak bedzie - kazdy Ford jest taki sam. Kazda Toyota tez taka sama, ale ... Toyota moze sie juz roznic parametrami od Forda.